### ISSUE TITLE
@slick-font-path not used in slick-theme.less

### ISSUE THREAD

**Issue by carbontwelve** [kenwheeler/slick#2228]
`@slick-font-path` not used in slick-theme.less. I installed via npm using the command `npm install slick-carousel --save` which installed 1.5.9. Having looked at the master here, I see the problem looks to be fixed, not sure why node is installing an older release?

====================================================================

#### Steps to reproduce the problem

Lines 25-32 in slick-theme.less do not use `@slick-font-path` variable:

```
@font-face{
    font-family: 'slick';
    font-weight: normal;
    font-style: normal;

    src: url('./fonts/slick.eot');
    src: url('./fonts/slick.eot?#iefix') format('embedded-opentype'), url('./fonts/slick.woff') format('woff'), url('./fonts/slick.ttf') format('truetype'), url('./fonts/slick.svg#slick') format('svg');
}
```

====================================================================

#### What is the expected behaviour?

The use of `@slick-font-path` to define the install path for the fonts.

====================================================================

#### What is observed behaviour?

The font paths are hard coded.

====================================================================

**Comment by simeydotme** [comment]
you're correct, fixed on master.

**Comment by riker09** [comment]
I came across this, as well. It it fixed on master, but why not in 1.5.9? Please forgive me, since I'm rather new to Bower, but shouldn't `bower install slick-carousel` install the latest, bug-free version? Right now it pulls the 1.5.9 tag, which still has the faulty paths.

**Comment by simeydotme** [comment]
haha @riker09 .... if there was a "bug-free" version, we could all go home and rest! :tada: 
Actually, Bower will only ever pull a `tag` unless you specifically tell it otherwise. This is because the `master` branch of a repo can generally have bleeding edge code which might not be safe. So although there's some issues with your `1.5.9` it's considered, mostly, safe.

There's going to be a release soon, I promise... either that or @kenwheeler is going to have a fucking rampant horde on his lawn ready to set ablaze his prized John Deere :fire::tractor: , shit in his mailbox :hankey::mailbox:  and kick over his Hillary Clinton signposts ! :footprints::dancer:
